pkgs/tools/system/rex/default.nix

```diff
@@ -44,6 +44,18 @@ perlPackages.buildPerlPackage rec {
 
   doCheck = false;
 
+  outputs = [ "out" ];
+
+  fixupPhase = ''
+    for sh in bash zsh; do
+      substituteInPlace ./share/rex-tab-completion.$sh \
+        --replace 'perl' \
+          "${pkgs.perl}/bin/perl -I ${pkgs.perlPackages.YAML}/${pkgs.perlPackages.YAML.perlModule.libPrefix}"
```

Comment:
`perl.withPackages (ps: [ ps.YAML ])` is more appropriate.

Reply:
thanks! fixed.